### PR TITLE
silent switch!!!!

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: RyuModManagerCLI/Program.cs [hunk 1]
```diff
@@ -97,9 +106,11 @@ public static async Task Main(string[] args)
                     Console.WriteLine("Unable to check for updates\n");
                 }
             }
-
+            if (isSilent == false)
```
Comment (mosamadeeb) [inline RyuModManagerCLI/Program.cs]: `if (!isSilent)`

File: RyuModManagerCLI/Program.cs [hunk 2]
```diff
@@ -97,9 +106,11 @@ public static async Task Main(string[] args)
...
+            {
             Console.WriteLine("Program finished. Press any key to exit...");
             Console.ReadKey();
```
Comment (mosamadeeb) [inline RyuModManagerCLI/Program.cs]: Missing indentation.

**Review by mosamadeeb — CHANGES_REQUESTED** [review]
Some small nits. Otherwise, looks good.

**Review by ghost — COMMENTED** [review]
Sounds great!

**Comment by ghost** [comment]
Fixed I believe??? https://github.com/GarnetSunset/RyuModManager/commit/e9c5e5523ef5365180d884c3329d3fc1b179e6e7

File: RyuModManagerCLI/Program.cs (hunk 2 above)
Comment (ghost) [reply RyuModManagerCLI/Program.cs]: https://github.com/GarnetSunset/RyuModManager/commit/604ec2fe852e5f9a93f327c8e9ec6a3e1cfe1c71

File: RyuModManagerCLI/Program.cs (hunk 1 above)
Comment (ghost) [reply RyuModManagerCLI/Program.cs]: https://github.com/GarnetSunset/RyuModManager/commit/e9c5e5523ef5365180d884c3329d3fc1b179e6e7

**Review by mosamadeeb — APPROVED** [review]

**Comment by mosamadeeb** [comment]
Looks fine. Thanks for the contribution!

**Pull request merged**